server/server.go

```diff
@@ -93,6 +94,11 @@ const (
 	rootDataDirPrefix = "network-runner-root-data"
 )
 
+func IsServerError(err error, serverError error) bool {
+	status := status.Convert(err)
+	return status.Code() == codes.Unknown && status.Message() == serverError.Error()
```

Comment:
It's not immediately obvious why these comparisons result in a server error, could you add some documentation?

Reply:
added:
```
// grpc encapsulates the non protocol-related, ANR server domain errors,
// inside grpc.status.Status structs, with status.Code() code.Unknown,
// and original error msg inside status.Message() string
// this aux function is to be used by clients, to check for the appropiate
// ANR domain error kind
```